### PR TITLE
Add default hidden setting

### PR DESCRIPTION
**Pull request by dale3h**
* Add default hidden setting for `config.json`
* Add support for `automation` entities
* Add support for `alert` entities

### CODE REVIEW

**Comment by keatontaylor** [comment]
Could you also update the customization section of the README.md file to reflect these changes.

**Comment by trisk** [comment]
"hidden_default" is kind of a confusing name. Maybe "default_hide_entities"?

**Comment by keatontaylor** [comment]
Why not follow the emulated hue naming scheme: expose_by_default: true

**Comment by dale3h** [comment]
Actually, I think that is perfect. I will try to get the changes made
tomorrow at the latest.
-- 
Dale Higgs
817-521-5544
dale3h@gmail.com

**Comment by dale3h** [comment]
Requested changes have been made, and then some :)

File: haaska.py [hunk 1]
```diff
@@ -295,10 +295,12 @@ def entity_domain(x):
...
     def is_supported_entity(x):
-        return entity_domain(x) in ha.config.allowed_domains
+        return entity_domain(x) in ha.config.exposed_domains
 
-    def is_skipped_entity(x):
+    def is_exposed_entity(x):
         attr = x['attributes']
+        if 'haaska_hidden' not in attr and 'hidden' not in attr:
+            return ha.config.expose_by_default
         return (('haaska_hidden' in attr and attr['haaska_hidden']) or
```
Comment (dale3h) [inline haaska.py]: I think this logic might be a bit off and is requiring the opposite value (`haaska_hidden: true`) for it to be exposed properly.

File: haaska.py [hunk 2]
```diff
@@ -295,10 +295,12 @@ def entity_domain(x):
...
                 ('hidden' in attr and attr['hidden']))
```
Comment (dale3h) [inline haaska.py]: I think this logic might be a bit off and is requiring the opposite value (`haaska_hidden: true`) for it to be exposed properly.

**Pull request merged**